packages/manager/modules/pci/src/projects/project/edit/remove/translations/Messages_en_GB.json

```diff
@@ -1,15 +1,6 @@
 {
   "pci_projects_project_edit_remove_title": "Delete project",
   "pci_projects_project_edit_remove_please_note": "Important: Please note that all of your services, resources, settings and data will be destroyed, without any option to restore at a later date. The only exception to this is failover IPs, which will be parked.",
-  "pci_projects_project_edit_remove_instances": "Your instances",
```

Comment:
Removed in https://github.com/ovh/manager/commit/1085cbe467e2e6e289a7238d29c85e3c2a8b0968#diff-bd9864564e725a8bb9d7b1ae3dc92263